MovieDB/MovieDB/Scenes/Dashboard/DashboardUseCase.swift

```diff
@@ -0,0 +1,15 @@
+import RxCocoa
+import RxSwift
+
+protocol DashboardUseCaseType {
+    func getMoviesFromCategory(_ category: CategoryType, page: Int) -> Observable<[Movie]>
```

Comment:
When declare functions of UseCase, you have to declare fully purpose of them, not general.
func getFirstPageMoviesOfCategory(category)
func getMoreMoviesOfCategory(category, page)
func refreshMoviesOfCategory(category)